webapp/static/interop-data.json

```diff
@@ -413,7 +413,7 @@
                 "description": "Forms",
                 "mdn": "https://developer.mozilla.org/docs/Web/HTML/Element/form",
                 "spec": "https://html.spec.whatwg.org/multipage/forms.html#the-form-element",
-                "tests": "/results/?label=experimental&label=master&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2023-forms",
+                "tests": "/results/?label=master&label=experimental&product=chrome&product=firefox&product=safari&aligned&view=interop&q=label%3Ainterop-2022-forms%20or%20label%3Ainterop-2023-forms",
                 "countsTowardScore": true
             },
             "interop-2023-grid": {
```

Comment:
Grid needs the same treatment as Flexbox too

Reply:
Done!